main.go

```diff
@@ -10,34 +10,43 @@ import (
 	"github.com/sul-dlss-labs/rialto-derivatives/actions"
 	"github.com/sul-dlss-labs/rialto-derivatives/derivative"
 	"github.com/sul-dlss-labs/rialto-derivatives/message"
+	"github.com/sul-dlss-labs/rialto-derivatives/repository"
+	"github.com/sul-dlss-labs/rialto-derivatives/runtime"
 	"github.com/sul-dlss-labs/rialto-derivatives/transform"
 )
 
 // Handler is the Lambda function handler
 func Handler(ctx context.Context, snsEvent events.SNSEvent) {
 	// stdout and stderr are sent to AWS CloudWatch Logs
 	log.Printf("Running Lambda function\n")
```

Comment:
Not a blocker on this, but I'm wondering if we want to either remove this (since there is already a "START" and "END" in the logs, or if we want to expand it to clarify which lambda is running (if that data is lost later when we figure out log aggregation).